### DISCUSSION TITLE
Cannot build CI on Xcode Cloud

### DISCUSSION THREAD

**Discussion by ngocdevv** [invertase/react-native-firebase#7999] [General]
I'm having trouble building an app on Xcode Cloud with a project that has a firebase app
Invalid `Podfile` file: No such file or directory @ rb_sysopen - /Volumes/workspace/repository/ios/node_modules/@react-native-firebase/app/ios_config.sh.

"@react-native-firebase/app": "^20.4.0",

**Comment by pehagg** [comment]
Having the same issue. The path for the file is off by one folder for some reason. Doesn't happen when I run `pod install` in the ios folder but running `bundle exec pod install --project-directory=ios` in the project root exposes the issue.

**Reply by ngocdevv** [reply]
@pehagg This is my solution:
bash /Volumes/workspace/repository/ios/node_modules/@react-native-firebase/app/ios_config.sh
chmod +x /Volumes/workspace/repository/ios/node_modules/@react-native-firebase/app/ios_config.sh
[script.txt](https://github.com/user-attachments/files/16845240/script.txt)

**Reply by pehagg** [reply]
This started happening for us after upgrading to RN 0.75.2. Downgrading back to 0.74.5 removes the issue. There's something funky with the latest RN version that messes up `pod install --project-directory=iOS` (with or without bundle).

**Reply by ngocdevv** [reply]
I am also having the problem you are mentioning on rn version 0.75.2. And I use the solution above to solve the problem

**Reply by pehagg** [reply]
Think we'll stay on 0.74 until this is resolved with a proper fix. Don't want to use workarounds and the upgrade can wait.

**Reply by pehagg** [reply]
Nonetheless, thanks for sharing your solution 👍🏻

**Comment by tellodaniel** [comment]
I had this problem in Xcloud and fixed it with this 

`ios/ci_scripts/ci_post_clone.sh`

```
#!/bin/sh

brew install node
brew install cocoapods
brew install yarn
yarn install

cd ../../node_modules/@react-native-firebase/app
chmod 755 ios_config.sh

cd ../../..
cd ios
pod install
```
